### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by bardyuzha**
https://bardyuzha.github.io/layout_html-form/
https://bardyuzha.github.io/layout_html-form/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,131 @@
...
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="single-fieldset">
+        <legend>Personal information</legend>
+
+        <label class="single-label">
+          Surname:
+          <input
```
Comment (mykhalenych) [inline src/index.html:22]: Screenshot 2023-04-24 at 17 20 38
check all tags

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,131 @@
...
+          Password:
+          <input type="password" name="password" required>
+        </label>
+
+      </fieldset>
+
+      <fieldset class="single-fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <span class="single-label inline">Do you love cats?</span>
```
Comment (mykhalenych) [inline src/index.html]: it is redundant add text inside label

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,131 @@
...
+
+        <label class="single-label inline">
```
Comment (mykhalenych) [inline src/index.html]: add wrap label into two inputs with type radio

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,161 @@
...
+            value="yes"
+          > Yes
+
+          <input
+            type="radio"
+            name="love_cats"
+            value="no"
+          > No
+
+        </label>
```
Comment (VitaliyBondarenko1982) [inline src/index.html]:
```suggestion
        <div class="single-label">Do you love cats?
          <label>
            <input
              type="radio"
              name="love_cats"
              value="yes"
            > 
            Yes
          </label>
          
          <label>
            <input
              type="radio"
              name="love_cats"
              value="no"
            > 
            No
          </label>
        </div>
```
each input should be wrapped in label.
without it click on Do you love cats or benween inputs or eveb on No select first radio input
and with this refactoring will be redundant inline class

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**